.github/workflows/standalone.yml

```diff
@@ -7,87 +7,86 @@ on:
     types: [opened, synchronize, reopened]
 jobs:
   build:
-    name: GALAHAD/${{ matrix.os }}/${{ matrix.compiler }}
+    name: GALAHAD/${{ matrix.os }}/${{ matrix.compiler }}/${{ matrix.version }}
     strategy:
       fail-fast: false
       matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        version: [10, 11, 12]
```

Comment:
```suggestion
        gcc-version: [10, 11, 12]
```

Reply:
Jari, we need the same version name for Intel / GNU compilers for the name of the builds:
`name: GALAHAD/${{ matrix.os }}/${{ matrix.compiler }}/${{ matrix.version }}`

Reply:
What do you think of `compiler_version`?

Reply:
Ah I see, sure `compiler_version` seems as good a name as any!